apps/designer/app/designer/features/sidebar-left/lib/header.tsx

```diff
@@ -2,33 +2,44 @@ import { Flex, IconButton, Text } from "@webstudio-is/design-system";
 import { Separator } from "@webstudio-is/design-system";
 import { Cross1Icon } from "@webstudio-is/icons";
 
-type HeaderProps = {
+type BaseHeaderProps = {
   title: string;
-  isClosable?: boolean;
-  onClose?: () => void;
+  actions: React.ReactNode;
 };
 
-export const Header = ({ title, isClosable = true, onClose }: HeaderProps) => {
+export const BaseHeader = ({ title, actions }: BaseHeaderProps) => {
```

Comment:
Here are suggestions:
1. lets rename actions to "suffix". We started using this pattern in design system, its more generic, so you can pass there anything you want, not only actions
2. lets rename BaseHeader to Header and delete the current Header
3. Lets export frequently used buttons to be composed with the header, e.g. the CloseButton

so the typical header with close button would look like this:

```
<Header suffix={<CloseButton onClick={...}/>}>
```